src/main/kotlin/uk/gov/justice/digital/hmpps/riskprofiler/services/ExtremismDecisionTreeService.kt

```diff
@@ -18,11 +17,11 @@ class ExtremismDecisionTreeService(private val repository: PathfinderService) {
   private fun decisionProcess(
     nomsId: String,
     previousOffences: Boolean,
-    pathFinder: Optional<PathFinder>
+    pathFinder: PathFinder?
   ): ExtremismProfile {
     val extremism = ExtremismProfile(nomsId = nomsId, provisionalCategorisation = RiskProfile.DEFAULT_CAT)
-    pathFinder.ifPresent { pf: PathFinder ->
-      val banding = pf.pathFinderBanding
+    if (pathFinder != null) {
```

Comment:
very minor but idiomatic Kotlin would have been pathFinder?.also {}

Reply:
ok, will add for next time